[2024-05-12T07:32:26.075+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-12T07:32:26.098+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: snowflake_crud_example.delete_data_table manual__2024-05-12T07:31:56.504338+00:00 [queued]>
[2024-05-12T07:32:26.109+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: snowflake_crud_example.delete_data_table manual__2024-05-12T07:31:56.504338+00:00 [queued]>
[2024-05-12T07:32:26.109+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-05-12T07:32:26.126+0000] {taskinstance.py:2327} INFO - Executing <Task(SnowflakeOperator): delete_data_table> on 2024-05-12 07:31:56.504338+00:00
[2024-05-12T07:32:26.134+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=91) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-12T07:32:26.135+0000] {standard_task_runner.py:63} INFO - Started process 93 to run task
[2024-05-12T07:32:26.135+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'snowflake_crud_example', 'delete_data_table', 'manual__2024-05-12T07:31:56.504338+00:00', '--job-id', '520', '--raw', '--subdir', 'DAGS_FOLDER/crud_table.py', '--cfg-path', '/tmp/tmphx3tvvks']
[2024-05-12T07:32:26.137+0000] {standard_task_runner.py:91} INFO - Job 520: Subtask delete_data_table
[2024-05-12T07:32:26.187+0000] {task_command.py:426} INFO - Running <TaskInstance: snowflake_crud_example.delete_data_table manual__2024-05-12T07:31:56.504338+00:00 [running]> on host 3078ffe3180a
[2024-05-12T07:32:26.302+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='snowflake_crud_example' AIRFLOW_CTX_TASK_ID='delete_data_table' AIRFLOW_CTX_EXECUTION_DATE='2024-05-12T07:31:56.504338+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-12T07:31:56.504338+00:00'
[2024-05-12T07:32:26.304+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-12T07:32:26.304+0000] {sql.py:276} INFO - Executing: 
delete from dbt_snowflake.public.customer 
[2024-05-12T07:32:26.317+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn' for task execution.
[2024-05-12T07:32:26.327+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn' for task execution.
[2024-05-12T07:32:26.328+0000] {connection.py:386} INFO - Snowflake Connector for Python Version: 3.7.1, Python Version: 3.12.2, Platform: Linux-5.15.146.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-05-12T07:32:26.329+0000] {connection.py:1211} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-05-12T07:32:26.330+0000] {connection.py:1221} INFO - THIS CONNECTION IS IN INSECURE MODE. IT MEANS THE CERTIFICATE WILL BE VALIDATED BUT THE CERTIFICATE REVOCATION STATUS WILL NOT BE CHECKED.
[2024-05-12T07:32:26.567+0000] {ssl_wrap_socket.py:100} INFO - THIS CONNECTION IS IN INSECURE MODE. IT MEANS THE CERTIFICATE WILL BE VALIDATED BUT THE CERTIFICATE REVOCATION STATUS WILL NOT BE CHECKED.
[2024-05-12T07:32:27.071+0000] {cursor.py:1032} INFO - query: [ALTER SESSION SET autocommit=False]
[2024-05-12T07:32:27.469+0000] {cursor.py:1045} INFO - query execution done
[2024-05-12T07:32:27.470+0000] {cursor.py:1205} INFO - Number of results in first chunk: 1
[2024-05-12T07:32:27.471+0000] {sql.py:457} INFO - Running statement: delete from dbt_snowflake.public.customer, parameters: None
[2024-05-12T07:32:27.471+0000] {cursor.py:1032} INFO - query: [delete from dbt_snowflake.public.customer]
[2024-05-12T07:32:28.694+0000] {cursor.py:1045} INFO - query execution done
[2024-05-12T07:32:28.696+0000] {sql.py:466} INFO - Rows affected: 5
[2024-05-12T07:32:28.699+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/providers/snowflake/hooks/snowflake.py:393: AirflowProviderDeprecationWarning: Call to deprecated method _make_common_data_structure. (The `_make_serializable` method is deprecated and support will be removed in a future version of the common.sql provider. Please update the DbApiHook's provider to a version based on common.sql >= 1.9.1.)
  result = self._make_common_data_structure(handler(cur))  # type: ignore[attr-defined]

[2024-05-12T07:32:28.702+0000] {snowflake.py:402} INFO - Rows affected: 5
[2024-05-12T07:32:28.703+0000] {snowflake.py:403} INFO - Snowflake query id: 01b44624-0002-1c3a-0002-b4620013f022
[2024-05-12T07:32:28.704+0000] {cursor.py:1032} INFO - query: [COMMIT]
[2024-05-12T07:32:29.245+0000] {cursor.py:1045} INFO - query execution done
[2024-05-12T07:32:29.246+0000] {cursor.py:1205} INFO - Number of results in first chunk: 1
[2024-05-12T07:32:29.247+0000] {connection.py:734} INFO - closed
[2024-05-12T07:32:29.554+0000] {connection.py:740} INFO - No async queries seem to be running, deleting session
[2024-05-12T07:32:29.852+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-12T07:32:29.922+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=snowflake_crud_example, task_id=delete_data_table, execution_date=20240512T073156, start_date=20240512T073226, end_date=20240512T073229
[2024-05-12T07:32:29.978+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-12T07:32:29.998+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-12T07:32:30.001+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
